pkg/runcfanotify/runcfanotify.go

```diff
@@ -19,6 +19,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/fs"
```

Comment:
If we decide to apply this, please sync the changes with [container-hook](https://github.com/inspektor-gadget/inspektor-gadget/blob/df2743c1bcabbdaf18604e69b454153ab4bfcb18/pkg/container-hook/tracer.go#L254) as that's the one we are currently using by default.